### ISSUE TITLE
postinstall assumes that typings is installed, but it's a devDependency not a regular dependency

### ISSUE THREAD

**Issue by nertzy** [kaiza/tslint-jasmine-rules#1]
We made an empty project and then tried to install the npm package and it failed:

```
# npm install tslint-jasmine-rules --save-dev

> tslint-jasmine-rules@0.2.1 postinstall /Users/pivotal/workspace/fake/node_modules/tslint-jasmine-rules
> typings install

sh: typings: command not found
npm WARN fake@1.0.0 No description
npm WARN fake@1.0.0 No repository field.
npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/Cellar/node/7.0.0/bin/node" "/usr/local/bin/npm" "install" "tslint-jasmine-rules" "--save-dev"
npm ERR! node v7.0.0
npm ERR! npm  v3.10.8
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! tslint-jasmine-rules@0.2.1 postinstall: `typings install`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the tslint-jasmine-rules@0.2.1 postinstall script 'typings install'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the tslint-jasmine-rules package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     typings install
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs tslint-jasmine-rules
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls tslint-jasmine-rules
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/pivotal/workspace/fake/npm-debug.log
```

**Comment by kaiza** [comment]
Sorry just saw this as I didn't get a notification. I'll look into it. Would also welcome pull requests.

**Comment by kaiza** [comment]
Should be resolved in version 1.0.2
